step-automation-packages/step-automation-packages-manager/src/main/java/step/automation/packages/AutomationPackageManager.java

```diff
@@ -627,6 +635,7 @@ protected static class Staging {
         List<Plan> plans = new ArrayList<>();
         List<ExecutiontTaskParameters> taskParameters = new ArrayList<>();
         List<Function> functions = new ArrayList<>();
+        ResourceManager resourceManager = new LocalResourceManagerImpl(new File("resources", new ObjectId().toString()));
```

Comment:
I would maybe use a dedicated context name "ap_staging_resources" for instance in order to quickly identify the root cause of potential leaks in the future

Reply:
renamed as suggested in last commit